### ISSUE TITLE
Possible typo in HTTP API documentation

### ISSUE THREAD

**Issue by fade2black** [RasaHQ/rasa_core#1182]
**Rasa Core version**: 0.11.3
**Python version**: 3.5.2

**Operating system** (windows, osx, ...): Linux container (Docker)

**Issue**: Possible Documentation typo.

[RASA Core documentation](https://rasa.com/docs/core/server/#) says that in order to append the tracker state of the conversation with events we should use `POST /conversations/(str: sender_id)/tracker/events`.  However it responses with HTTP 500. On the other hand the source code on [RASA Core github](https://github.com/RasaHQ/rasa_core/blob/549786c000efd76cc3ea6b7bcc68b2c40c840534/rasa_core/server.py) has the following line (259)

```
@app.route("/conversations/<sender_id>/tracker/events",
               methods=['PUT'])
```
When I change `curl XPOST` to `curl XPUT` it works.

**Comment by fade2black** [comment]
@akelad 
This is how I run the server 
```
python -m rasa_core.run  --enable_api -d models/dialogue -u models/nlu/default/simplebotnlu --credentials credentials.yml -o out.log
```

This is how I post a request (using curl) and its response
```
root@2c925f179988:/# curl -XPOST http://172.17.0.2:5005/conversations/bayram/tracker/events -d '[{"event": "slot", "name": "name", "value": "Bayram"}, {"event": "action", "name": "action_listen"}]'
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>

root@2c925f179988:/#
```

This is an example from [your site](https://rasa.com/docs/core/server/#)
```
root@2c925f179988:/# curl -XPOST http://172.17.0.2:5005/conversations/default/tracker/events -d   '[{"event": "slot", "name": "cuisine", "value": "mexican"},{"event": "action", "name": "action_listen"}]'
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request.  Either the server is overloaded or there is an error in the application.</p>

root@2c925f179988:/#
```

**Comment by fade2black** [comment]
@akelad 
Thank you for reply.
So, that means in order to append, say, 5 events I need to call 5 times, right?
Is there any possibility to improve appending event so that we could append more than 1 events at a time? Should I open a new issue?  This is just an proposal.

**Comment by akelad** [comment]
No, we're gonna keep it that way, as this is the REST convention with `POST` to an endpoint, one item at a time to create a new resource in the collection.
So yes you have to call it 5 times :)

**Comment by fade2black** [comment]
One more question. In fact I asked it on RASA Forum about a couple of weeks ago but it is I guess still unanswered. 
In your HTTP API documentation I cannot see information about format of input to an endpoint and its response. You give only particular examples. But I think examples do not provide exhaustive information about input/response format. Response may vary depending on input.
Also the documentation says 'The format of the passed events is the same as for the `/continue` endpoint.', but I couldn't find where the `/continue` endpoint is described.

**Comment by akelad** [comment]
@fade2black thanks for mentioning that, I just checked the updated docs for the master branch and they're more specific about that as well. They have the openapi spec format now :) I'm gonna close this issue as it's all been updated, once this PR is merged (#1193) it will show up on the master version of the docs properly
